Libraries/LibWeb/CSS/Interpolation.cpp

```diff
@@ -586,6 +594,15 @@ NonnullRefPtr<CSSStyleValue const> interpolate_value(DOM::Element& element, Calc
             layout_node = *node;
         return CSSColorValue::create_from_color(interpolate_color(from.to_color(layout_node), to.to_color(layout_node), delta), ColorSyntax::Modern);
     }
+    case CSSStyleValue::Type::Edge: {
```

Comment:
`EdgeStyleValue::to_string()` has logic for converting its value to top/left-relative, which might be useful here. Otherwise I'm not sure this will interpolate a "from is left, to is right" situation correctly.

Reply:
I've split the logic that does this into its own method, which I now use when interpolating edge values.